cpp/turbodbc/Library/src/make_description.cpp

```diff
@@ -47,7 +47,7 @@ namespace {
 	using description_ptr = description const *;
 
 	struct description_by_value : public boost::static_visitor<description_ptr> {
-		description_ptr operator()(long const &) const
+		description_ptr operator()(intptr_t const &) const
```

Comment:
Also, since this deals with a 64bit integer parameter, I'd prefer `int64_t` here.